kedro/extras/datasets/pandas/gbq_dataset.py

```diff
@@ -164,8 +164,8 @@ def _validate_location(self):
 
         if save_location != load_location:
             raise DataSetError(
-                "`load_args['location']` is different from `save_args['location']`. "
-                "The `location` defines where BigQuery data is stored, therefore has "
+                """ "load_args['location']" is different from "save_args['location']". """
```

Comment:
```suggestion
                """"load_args['location']" is different from "save_args['location']". """
```
I think this should work without the initial space.